tests/test_config.py

```
@@ -375,51 +390,52 @@ def test_config_path_resolution_edge_cases(self):
         mock_args = MagicMock()
         mock_args.config = "../config/test.yaml"
 
-        paths = get_config_paths(args=mock_args)
+        with patch("mmrelay.paths.get_home_dir", return_value=Path("/tmp/home")):
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/home"

(S108)